subjects/ai/numpy/README.md

```diff
@@ -26,53 +13,52 @@ I suggest to use the most recent one.
 
 ### Resources
 
-- https://medium.com/fintechexplained/why-should-we-use-NumPy-c14a4fb03ee9
-- https://numpy.org/doc/
-- https://jakevdp.github.io/PythonDataScienceHandbook/
+- [Why Should We Use NumPy](https://medium.com/fintechexplained/)why-should-we-use-NumPy-c14a4fb03ee9
```

Comment:
There is a typo in this hyperlink - the parenthesis should close till the end

Reply:
done